generators/addons/templates/jest.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "preset": "@voitanos/jest-preset-spfx-react16",
+  "rootDir": "../src",
+  "coverageReporters": [
+    "text",
+    "json",
+    "lcov",
+    "text-summary",
+    "cobertura"
+  ],
+  "reporters": [
+    "default",
+    "jest-junit"
```

Comment:
@StfBauer here most of the file is already provided by the jest step, however I need to inject additional values (jest-junit) how do you suggest I address that?

Reply:
I ended up leveraging the end lifecycle event